README.md

```diff
@@ -172,7 +168,7 @@ You can deploy this project using:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- Add reducers and actions.
+
```

Comment:
- [ ] You did a great job! The only thing we need to fix it your `issues` link. Kindly fix the link and resubmit to get approved.

Reply:
@Rustamxon7  I already did it, thanks for your review! :handshake: